docs/source/index.rst

```diff
@@ -20,8 +20,8 @@ First, let's train a model with :class:`~sap.aibus.dar.client.flow.create.Create
     from sap.aibus.dar.client.workflow.model import ModelCreator
 
     creator = ModelCreator.construct_from_credentials(
```

Comment:
I think we can lead by example by here by removing the service key completely and show that we read it from a file (would just be a dummy example).

Or do you think we are fine if we simply add a comment here stating that one should never put the service key into the code and this is just for the purpose of illustrating the format?

Reply:
Hi @mhaas, I've updated the example here. Let me know if this looks better.